core-bundle/src/EventListener/DataContainer/DefaultOperationsListener.php

```diff
@@ -212,7 +212,7 @@ private function getToggleField(string $table): string|null
     private function isGranted(string $actionClass, string $table, DataContainerOperation $operation): bool
     {
         $subject = match ($actionClass) {
-            CreateAction::class => new CreateAction($table),
+            CreateAction::class => $this->copyAction($table, $operation),
```

Comment:
Please also adjust the `DefaultOperationsListenerTest` class accordingly.

Reply:
I mean adding a test that `$new['id']` is unset and `$new['sorting']` is set to the correct value.

Reply:
added in 31d8a12e99638c78e171bc4440e397a4b89fa619